src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/design/DesignForExtensionCheck.xml

```diff
@@ -128,6 +128,11 @@
                <description>Specify annotations which allow the check to
  skip the method from validation.</description>
             </property>
+            <property default-value=".*"
+                      name="requiredJavadocPhrase"
+                      type="java.util.regex.Pattern">
+               <description>Specify the comment text pattern which qualifies a</description>
```

Comment:
The description is incomplete. There may be a issue in the metadata generation/validation.

Reply:
@nmancus1 , please investigate problem to generate full description, it might be separate issue, but better to identify it now.

Reply:
I am struggling to find documentation for the metadata generator, can you point me in the right direction?

Reply:
@romani This is a separate issue and has been seen before. I pointed @nmancus1 to the fault of the error. I recommend an issue be created and this get fixed. It will completely baffle new users, and I question if admins involved in its creation now it or the solution to overecome it right now.

Reply:
there is no documentation on generator, please create issue on it , while we see it.
Is there a way to make workaround ?

Reply:
some other Checks do have long descriptions and it works fine for them.
in worse case I would agree to make description single line and suppress violation from LineLenght on it, until issue is resolved.